### PR TITLE
[PTQ] Register statistics for weights the same as for activations

### PR DESCRIPTION
**Pull request by kshpv**
### Changes

Introduce the way to register weight quantizers statistics the same for weight and for activations.

### Reason for changes

To vanish the problems with finding the weight tensor.

### Related tickets

102594

### Tests

Updated

### CODE REVIEW

File: nncf/experimental/openvino_native/graph/node_utils.py [hunk 1]
```diff
@@ -80,3 +80,15 @@ def get_node_with_bias_value(add_node: NNCFNode, nncf_graph: NNCFGraph) -> Optio
...
 
     return bias_constant if bias_constant.metatype == OVConstantMetatype else None
+
+
+def get_result_node_name(output_name: str, port_id: int) -> str:
+    """
+    Returns name of Result based on node name and its port.
+
+    :param output_name: Node name.
+    :param port_id: Node's output port.
```
Comment (daniil-lyakhov) [inline nncf/experimental/openvino_native/graph/node_utils.py:90]: This is not true because for `PRE_LAYER` insertion it's `port_id` of inputs

Comment (kshpv) [reply nncf/experimental/openvino_native/graph/node_utils.py]: Done

**Review by daniil-lyakhov — APPROVED** [review]
👍

**Review by alexsu52 — APPROVED** [review]
LGTM

**Review by KodiaqQ — APPROVED** [review]

**Pull request merged**